giles-eco/src/main/java/edu/asu/diging/gilesecosystem/web/rest/DigilibPassthroughController.java

```diff
@@ -41,7 +41,7 @@
     @Autowired
     private DigilibConnector digilibConnector;
 
-    @TokenCheck
+    @ImageAccessCheck
     @RequestMapping(value = "/rest/digilib")
     public ResponseEntity<String> passthroughToDigilib(
```

Comment:
actually, you can probably just inject the ```fn``` parameter into the method (I don't know why I didn't do that orginally). But since that's always required to get an image, that should work.